Assets/SequenceSDK/Ethereum/Tests/SequenceEthClientTests.cs

```diff
@@ -135,7 +135,7 @@ public async Task TestBlockRange()
         [TestCaseSource(nameof(chainIdCases))]
         public async Task TestChainId(Chain chain)
```

Comment:
TestChainId(TestnetRootPorcini) (0.226s)
---
```
Expected no exception, but got: File load exception: HTTP/1.1 404 Not Found reason: {"jsonrpc":"2.0","id":1,"error":{"code":-32600,"message":"no network named 'root-porcini' found"},"result":null}

Curl-equivalent request: curl -X POST 'https://nodes.sequence.app/root-porcini/AQAAAAAAAJNs_93iq6dLeDfnHJS0FYiekmU' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'X-Access-Key: AQAAAAAAAJNs_93iq6dLeDfnHJS0FYiekmU' -d '{"jsonrpc":"2.0","id":1,"method":"eth_chainId","params":[]}'

```

Reply:
Node gateway for Root Porcini doesn't appear to be live yet
https://status.sequence.info/